client/lib/utils/test/error.js

```diff
@@ -0,0 +1,102 @@
+import { expect } from 'chai';
+import {
+	isFieldError,

```

Comment:
Cuz copy paste from another test and cuz it allows to add more functions later one.